net/libreswan/files/etc/hotplug.d/ipsec/01-user

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
Shouldn't we use a separate hotplug directory here, `libreswan`? Because if someone also installs strongswan we will have a problem here.

Reply:
Done in https://github.com/openwrt/packages/pull/19233